sensing/position_estimation.hpp

```diff
@@ -201,10 +201,13 @@ class Position_estimation: public INS
 
     float gravity;                          ///< Value of the gravity
 
+    float barometer_bias;
```

Comment:
done in https://github.com/lis-epfl/MAVRIC_Library/pull/311/commits/2f813793cac5e2ebf2a7fd5f365261eca57dc003